blog/2024-05-20-v0-format-modifications.mdx

```diff
@@ -1,60 +1,62 @@
 ---
```

Comment:
This file should be renamed to reflect the new publication date (vs `2024-05-20`)